2-Authorization/0-call-api-vanillajs/SPA/public/authConfig.js

```diff
@@ -9,6 +9,7 @@ const msalConfig = {
         authority: 'https://Enter_the_Tenant_Subdomain_Here.ciamlogin.com/', // Replace the placeholder with your tenant name
         redirectUri: '/', // You must register this URI on Microsoft Entra admin center/App Registration. Defaults to window.location.href e.g. http://localhost:3000/,
         postLogoutRedirectUri: '/', // Indicates the page to navigate after logout.
+        authority: 'https://login.microsoftonline.com/Enter_Your_Tenant_Id_Here',
```

Comment:
Line 9 & Line 12 are duplicated, seems confusing to users. Recommend we keep 12 only